docs/file-formats/xml-data-file.md

```diff
@@ -980,6 +945,47 @@ Technical note: A `<link>` is in fact a `<resource restype="LinkObj">`. But it i
 shortcut, so that the XML file can be validated more precisely.
 
 
+### `<video-segment>` / `<audio-segment>`
+
+`<video-segment>` and `<audio-segment>` represent sections of an audio/video file. 
+They can have the following predefined properties:
+
+- `isSegmentOf` (1)
+- `hasSegmentBounds` (1)
+- `hasTitle` (0-1)
+- `hasDescription` (0-n)
+- `hasComment` (0-n)
+- `hasKeyword` (0-n)
+- `relatesTo` (0-n)
+
+The bounds of the segment are defined by two floating point numbers, which represent the seconds.
+The segment from 0h 10min 00s to 0h 20min 00s would thus be represented by the interval `600:1200`.
+
+Example:
+
+```xml
+<video-segment label="Segment of video_thing_1" id="video_segment_0" permissions="res-default">
+    <resptr-prop name="isSegmentOf">
+        <resptr permissions="prop-default">video_thing_1</resptr>
+    </resptr-prop>
+    <interval-prop name="hasSegmentBounds">
+        <interval permissions="prop-default">10:20</interval>  # from 0h 00min 10s to 0h 00min 20s
+    </interval-prop>
+    <text-prop name="hasComment">
+        <text encoding="xml" permissions="prop-default">Optional comment</text>
+    </text-prop>
+</video-segment>
+```
+
+Technical notes: 
```

Comment:
I don't think this is the right place for this information. It is basically only relevant for dsp-tools devs and will confuse everyone else.

Reply:
I see your point. The same applies for the `<link>` and `<region>` and so on, some lines above. We could consider placing this information somewhere else, but I don't know where. I feel it should be documented somewhere, and the docs just seemed the most convenient place (at least for me).